docs/application/native/api/iot-headed/5.5/group__Evas__Object__Group__Basic.html

```diff
@@ -402,7 +403,7 @@
 <a class="el" href="group__Evas__Object__Group.html#ga21e8604c0e4a93e469bff4bd069e82b5">evas_object_resize()</a> </dd>
 <dd>
 <a class="el" href="group__Evas__Object__Group__Basic.html#ga6f52fab1a96dc58c85ee481d84bac871">evas_object_geometry_get</a></dd></dl>
-<dl class="since"><dt><b>Since:</b></dt><dd>1.8 </dd></dl>
+<dl class="since"><dt><b>Since:</b></dt><dd>1.8  3.0f(2.3.1) </dd></dl>
```

Comment:
Would you please check? Several errors like this exist in this page.

![image](https://user-images.githubusercontent.com/9958374/102976503-fd052e80-4544-11eb-803d-1b8cc1461cdc.png)

Reply:
checked

Reply:
https://review.tizen.org/gerrit/#/c/platform/upstream/efl/+/250985/